packages/main/src/components/ActionSheet/ActionSheet.stories.tsx

```diff
@@ -24,7 +24,7 @@ const meta = {
     header: {
       control: { disable: true }
     },
-    a11yConfig: { table: { category: 'A11y props' } }
+    accessibilityAttributes: { table: { category: 'A11y props' } }
```

Comment:
```suggestion
    accessibilityAttributes: { table: { category: 'Accessibility props' } }
```
Should we avoid the abbreviation here?